alwaysontop/render/index.js

```diff
@@ -79,6 +90,12 @@ class AlwaysOnTop extends EventEmitter {
 
     _onConferenceJoined() {
         logInfo('on conference joined');
+        if (this._joined) {
```

Comment:
Why is this logic necessary?

Reply:
cause _onConferenceJoined is called twice when showOnPrejoin, as prejoinScreenLoaded and videoConferenceJoined both trigger it.

Reply:
Also breakoput rooms, but what is the issue?

Reply:
I thought to avoid unexpected issues by it being called twice, but I'll test without it as it might be fine without the check.

Reply:
👍

Reply:
@saghul retested without the flag and it's not working as expected, it actually revealed an existing issue on AOT (without my changes). Seems like ipcRenderer event handler is not cleaned properly on leave if `ipcRenderer.on(...` is called multiple times.

Reply:
good catch